dev/import-images-k3d

```diff
@@ -12,3 +12,8 @@ if [ "$upstream_ctx" != "$downstream_ctx" ]; then
 else
   echo "not importing agent to any downstream clusters. Set FLEET_E2E_CLUSTER_DOWNSTREAM"
 fi
+
+# Build and import git server image
+cd e2e/assets/gitrepo
```

Comment:
I run this script a lot while debugging. Can we make gitserver build optional? Maybe via an env var?

Reply:
Good point, I've added support for env var `FORCE_GIT_SERVER_BUILD` to rebuild the image even if it already exists. Otherwise it should only be built once on your machine.